_acs-aem-tools/features/clientlibs-optimizer/index.md

```diff
@@ -15,7 +15,13 @@ To get to the AEM Tools console from the Touch UI, from the left rail navigation
 
 ## Overview
 
-Client Library Optimizer is a tool that accepts a list client library categories and derives the embed rule so they can all be compacted into a single include, reducing the # of HTTP requests required (ideally, to one).
+Client Library Optimizer is a tool that accepts a list of client library categories and derives the embed rule so they can all be compacted into a single include, reducing the # of HTTP requests required (ideally, to one).
 
 ![AEM Clientlibs Optimizer](images/screenshot.png)
 
+## Technical Background
+
+Client libraries can have *dependencies* and *embeds*. The difference between those two concepts is that dependencies lead to additional includes/requests, while embeds are contained in the original client library's response (i.e. are concatenated with the JS/CSS of the client library itself). However, transitive embeds are not considered by AEM (tracked in GRANITE-2288), e.g. if client library `a` embeds client library `b` and `b` embeds client library `c`, `c` is not contained in the response for `a`. Dependencies of embedded libraries are not considered by AEM either (tracked in GRANITE-1416).
```

Comment:
I would remove these ticket IDs, as they are not publicly available.

Reply:
Although not publicly available those references are mentioned in the AEM release notes in case the according issues are fixed. That helps to identify future AEM versions which do no longer suffer from these limitations.